frontend/libs/studio-components/src/components/StudioRecommendedNextAction/StudioRecommendedNextAction.module.css

```
@@ -0,0 +1,11 @@
+.buttonGroup {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-start;
+  gap: 12.5px;
```

Maybe we can use a spacing token from the design system instead? 😊 
```suggestion
  gap: var(--fds-spacing-3)
```